modules/codegen/src/main/scala/com/twilio/guardrail/generators/SwaggerGenerator.scala

```diff
@@ -56,31 +60,31 @@ object SwaggerGenerator {
         } yield className
 
       case GetParameterName(parameter) =>
-        Target.fromOption(Option(parameter.getName()), "Parameter missing \"name\"")
+        Target.fromOption(Option(parameter.getName()), s"Parameter missing 'name': ${parameter}")
```

Comment:
swagger-parser's models log each entry with `null` fields and whatnot. I don't know if `parameter`s are `Schema[_]`s, but https://github.com/twilio/guardrail/blob/d6f2dbb74d37cd8374b0c9304bde8f4e4a6bc272/modules/codegen/src/main/scala/com/twilio/guardrail/terms/SwaggerTerms.scala#L58 may be useful here

Reply:
`Parameter`s are not `Schema[_]`s, as it turns out.

Reply:
(And they only have 7 fields, so I guess it's no big deal if we log nulls.)